src/components/Header/index.scss

```diff
@@ -1,4 +1,6 @@
 .easi-header {
+  border-bottom: 1px solid #979797;
```

Comment:
Have the designs changed? I specifically remember leaving this style off the the header because the header did not always have border on the bottom. IIRC, the mobile views do not have a border on the bottom and pages WITHOUT the system nav list also do not have border on the bottom.

Reply:
Maybe they have changed... I looked through all the designs here https://www.figma.com/file/JZ2P1eqFM6IHrRmXFRMHWl/EASi-Wires?node-id=954%3A1573 (all the For Eng designs) and they all have the bottom border. Are you looking at a different reference point or have the designs changed?

Reply:
IIRC, there is no border when the user is logged out. I'm having a difficult time finding that spec in Figma. I'll confirm with design whether there should _always_ be a border.

For the record, I think there should always be a border.